docs/evs/evs.md

```diff
@@ -0,0 +1,72 @@
+# Huawei Cloud EVS-CSI Plugin
+
+EVS disk types and features are provided to help you quickly select your desired disk type to match services. 
+This facilitates rapid service migration to Huawei Cloud.
+
+After purchasing a data disk, you must attach the disk to a server and initialize (format) it before using it. 
+You can expand, detach, or delete your disk any time based on service requirements; 
+easily manage your disks and snapshots; and view monitoring metrics in real time to learn about the health status of your disks.
+
+## Compatibility
+
+For sidecar version compatibility , please refer compatibility matrix for each sidecar here 
+-https://kubernetes-csi.github.io/docs/sidecar-containers.html
+
+
+## Deploy
+
+### Prerequisites
+
+- docker, kubeadm, kubelet and kubectl has been installed.
+
+### Steps
+- Create the config file according to
+
+The driver initialization depends on a [cloud config file](../../deploy/cloud-config). Make sure it's in `/etc/evs/cloud-config` on your node.
+
+Click to view the detailed description of the file: [cloud-config](../cloud-config.md)
+
+- Create RBAC resources.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml
+kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-node.yaml
+kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-secret.yaml
+```
+
+- Create HuaweiCloud EVS CSI driver deamonset

```

Comment:
suggest to use "Install HuaweiCloud EVS CSI Plugin"

Reply:
done